Lets Do This/Controllers/Base/LDTTabBarController.m

```diff
@@ -63,15 +63,27 @@ - (void)viewDidAppear:(BOOL)animated {
         [destNavVC styleNavigationBar:LDTNavigationBarStyleClear];
         [self presentViewController:destNavVC animated:YES completion:nil];
     }
+    else {
+        if ([DSOUserManager sharedInstance].activeCampaigns.count == 0) {
+            [[DSOUserManager sharedInstance] loadCurrentUserAndActiveCampaignsWithCompletionHander:^(NSArray *activeCampaigns) {
+                [[NSNotificationCenter defaultCenter] postNotificationName:@"activeCampaignsLoaded" object:self];
+            } errorHandler:^(NSError *error) {
+                [[NSNotificationCenter defaultCenter] postNotificationName:@"epicFail" object:self];
+            }];
+        }
+    }
 }
 
 #pragma mark - LDTTabBarController
 
-- (void)loadCurrentUserAndCampaigns {
+- (void)reloadCurrentUser {
     UINavigationController *initialVC = (UINavigationController *)self.viewControllers[0];
     [initialVC popToRootViewControllerAnimated:YES];
-    LDTCauseListViewController *causeListViewController = (LDTCauseListViewController *)initialVC.topViewController;
-    [causeListViewController loadCurrentUserAndCampaigns];
```

Comment:
Don't need to reload the campaigns again - they've already been loaded. We can just sync the new current user instead.